processors/src/SvtRawDataAnaProcessor.cxx

```diff
@@ -251,46 +261,156 @@ void SvtRawDataAnaProcessor::initialize(TTree* tree) {
 
 
 bool SvtRawDataAnaProcessor::process(IEvent* ievent) {
+    //std::cout<<"hello5"<<std::endl;
     Float_t TimeRef=-0.0;
     Float_t AmpRef=1000.0;
     double weight = 1.;int count1=0;int count2=0;
     long eventTime = evH_->getEventTime();
     
     //I AM DOING CLUSTER MATCHING HERE :)
-    //std::cout<<"Here is the eventTime"<<eventTime<<std::endl;
+    std::cout<<"Here is the eventTime"<<eventTime<<std::endl;
     //std::cout<<"Here is the TSBank Trigger Time"<<tsBank_->T<<std::endl;
     bool doClMatch = true;
     
-    
+    int STR = -10000;
+    int HITC = 0;
+    int HITL = 0;
+    //int MissHit = 0;
+    float otherTime = 69420.0;
     //if((doClMatch)and(not((tsBank_->prescaled.Single_3_Top==1)or(tsBank_->prescaled.Single_3_Bot==1)))){return true;}
     
     
     //ONLY POSITRONS, MAY USE FEE's 
     //ONCE I DETERMINE A CLUSTER WHICH IS IN LINE WITH TRIG, I CAN USE ANY CLUSTERS CLOSE IN TIME.
     
     //std::cout<<"Trigger Time: "<<vtpBank_->singletrigs.at(0).T<<std::endl;
+    
+    
+    //std::cout<<"I got here 9B"<<std::endl;
```

Comment:
remove all old debug statements, unless they might be needed in the future

Reply:
Done